examples/server-example.py

```diff
@@ -88,12 +88,8 @@ def run(self):
     #server.set_endpoint("opc.tcp://localhost:4840/freeopcua/server/")
     server.set_endpoint("opc.tcp://0.0.0.0:4840/freeopcua/server/")
     server.set_server_name("FreeOpcUa Example Server")
-    # set possible endpoint policies for clients to connect through
-    server.set_security_policy(["None",
-                                "Basic128Rsa15_Sign",
-                                "Basic128Rsa15_SignAndEncrypt", 
-                                "Basic256_Sign",
-                                "Basic256_SignAndEncrypt"])
+    # set all possible endpoint policies for clients to connect through
+    server.set_security_policy([0, 1, 2, 3, 4])
```

Comment:
that is not correct way to use enums ;-)

    server.set_security_policy([ua.SecurityPolicies.NoSecurity, ua.SecurityPolicies.Basic128Rsa15_Signm etc...])

Reply:
I see. Just tried to decrease the number of letters written :)
Is the SecurityPolicyType class definition fine?